api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java

```diff
@@ -72,18 +75,42 @@ public Trace() {
     }
 
     /**
-     * @return the id
+     * @return the traceId
+     */
+    public String getTraceId() {
+        return traceId;
+    }
+
+    /**
+     * This method sets the trace id. When the fragment and trace id
```

Comment:
@jpkrohling What you are your thoughts - I'm happy to go with the majority decision, but personally I mainly focus on the comments on the methods when looking at javadocs, and therefore might miss a field comment just because of being lazy and not navigating the link :)